<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMMyMusic</class>
 <widget class="QWidget" name="SMMyMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>840</width>
     <height>580</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="tools" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QToolButton" name="tbShangchuan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbFasong">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>4</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShanchu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>4</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAllSelect">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>5</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShuaxin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>4</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
